src/main/java/com/crowdin/cli/commands/functionality/ObsoleteSourcesUtils.java

```diff
@@ -29,7 +29,10 @@ public static Map<String, File> findObsoleteProjectFiles(
 
     private static boolean checkExportPattern(String exportPattern, File file) {
         String fileExportPattern = ProjectFilesUtils.getExportPattern(file.getExportOptions());
-        return exportPattern.equals(fileExportPattern != null ? Utils.normalizePath(fileExportPattern) : null);
+        if (fileExportPattern == null) {
```

Comment:
@andrii-bodnar What do you think of this approach?

Reply:
Does this mean that files with the `null` export pattern will be considered obsolete and deleted, or the opposite?

Reply:
It basically uses all projectFiles (files received by the api) and checks for each file:
- if that file matches the config patterns (source-pattern, ignore and preserveHierarchy)
- if it matches the exportPattern (if the api returns null for the export pattern we ignore it now by this change)
- and if the file has `isFileDontHaveUpdate=true` (which seems to check if the file has not been updated previously by the executed cli cmd as well)

Reply:
I adjusted some var and method names so they match the rest of the code and are hopefully more descriptive.